src/scenes/finalScene.js

```diff
@@ -0,0 +1,72 @@
+import Phaser from "../lib/phaser.js";
+
+let config = {
+    type: Phaser.AUTO,
+    width: 1920,
+    height: 1080,
+    physics: {},
+    scene: {
+        preload: preload,
+        create: create
+    }
+}
+
+let game = new Phaser.Game(config)
+
+function preload() {
+    this.load.image('pool', '../assets/images/pond.png');
```

Comment:
Tu peux ajouter `this.load.setBaseURL("../assets");`
pour éviter de mettre `../assets` partout